velox/functions/prestosql/tests/ArraySortTest.cpp

```diff
@@ -583,6 +583,22 @@ TEST_F(ArraySortTest, lambda) {
       "(x, y) -> if(length(x) < length(y), 1, if(length(x) = length(y), 0, -1))");
 }
 
+TEST_F(ArraySortTest, failOnMapTypeSort) {
+  auto data = makeRowVector({makeDataArray<TestMapType>({2, 1, 0})});
+  static const StringView kErrorMessage =
+      "Scalar function signature is not supported"_sv;
+  auto testFail = [&](const std::string& name, const std::string& lambdaExpr) {
+    VELOX_ASSERT_THROW(
+        evaluate(fmt::format("{}(c0, {})", name, lambdaExpr), data),
+        kErrorMessage);
+    VELOX_ASSERT_THROW(
+        evaluate(fmt::format("{}(c0)", name), data), kErrorMessage);
+  };
+
+  testFail("array_sort", "x -> x");
```

Comment:
Can we also test non-lambda versions of the functions?

Reply:
The function testFail test both non-lambda and lambda version.
```C++
VELOX_ASSERT_THROW(
        evaluate(fmt::format("{}(c0)", name), data), kErrorMessage);
```

Reply:
Got it. I missed this. Thank you for clarifying.